compiler/rustc_borrowck/src/session_diagnostics.rs

```diff
@@ -5,6 +5,17 @@ use rustc_span::Span;
 
 use crate::diagnostics::RegionName;
 
+#[derive(Diagnostic)]
+#[diag(borrowck_cannot_use_when_mutably_borrowed, code = "E0503")]
+pub(crate) struct CannotUseWhenMutablyBorrowed<'a> {
+    #[primary_span]
+    pub span: Span,
+    #[label]
```

Comment:
There is no corresponding `label` in the ftl file.

Reply:
I am still trying to understand how to use them, would just including the rest of the parameters under primary span work ? I can then just put their placeholders in the ftl file. I read the documentation but I was not really sure how to incorporate them in the .ftl file

Reply:
Adding a `#[label]` is a reference to a `.label = <msg>` attribute on the `borrowck_cannot_use_when_mutably_borrowed` message, so it should have something like:

```
borrowck_cannot_use_when_mutably_borrowed = cannot use `{$desc}` when mutably borrowed
    .label = <msg>
```